client/src/Asset.tsx

```diff
@@ -133,6 +151,32 @@ function Asset(): JSX.Element {
 
     const bigImg = asset.imgUrls[selectedImgInd];
 
+    const showMiniImgs =
+        asset.imgUrls.map((imgUrl, ind) => {
+          if (ind !== selectedImgInd) {
+            return <img
+                key={imgUrl + `_${ind}`}
```

Comment:
sorry, i'm going to change my mind again
in the future i'm planning for us, we should never have duplicate image urls in this array
so let's keep the key as just `imgUrl` and let's make the fix at the source. aka remove the duplicate image from the dummy data source
if that doesn't work for any reason just keep this in here and we can figure out down the line